CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ endif()
 
 if(POLICY CMP0026)
     # Allow use of the LOCATION property
-    cmake_policy(SET CMP0026 OLD)
+    cmake_policy(SET CMP0026 NEW)
```

Comment:
NEW is default so you may omit this block now.

Reply:
Actually if I read the documentation right, the default is always OLD unless you have a `cmake_policy(VERSION major.minor[.patch[.tweak]])` command?